packages/jellyfish-block/src/block.ts

```diff
@@ -0,0 +1,48 @@
+import { BlockHeader, CBlockHeader } from './blockHeader'
+import { BufferComposer, ComposableBuffer } from '@defichain/jellyfish-buffer'
+import {
+  TransactionSegWit,
+  Transaction,
+  CTransaction,
+  CTransactionSegWit
+} from '@defichain/jellyfish-transaction'
+import { SmartBuffer } from 'smart-buffer'
+
+export interface Block {
+  blockHeader: BlockHeader // -------------| 124 bytes + signature (n = VarUInt{1-9 bytes} + n bytes)
+  transactions: Array<TransactionSegWit | Transaction> // ----| n = VarUInt{1-9 bytes}, + n transactions, list of transactions
```

Comment:
we need a `Block { blockHeader, transactions<Transaction[]> }` test for document as well

Reply:
There is a test block with  `describe('block with transaction as Transaction'` in block.test.ts now.
Is that what you mean ?

Reply:
yep.. see that now